git-remote-gosh/src/lib.rs

```diff
@@ -12,6 +12,9 @@ extern crate serde_json;
 extern crate base64;
 extern crate base64_serde;
 
+#[macro_use]
+extern crate derive_builder;
```

Comment:
How is this used?

Reply:
TLDR: autogenerate xBuilder for x struct